### PR TITLE
removes standard borg

### PR DESCRIPTION
**Pull request by hornygranny**
removes standard borg

i wanted to remove just the baton but i hadn't realized how totally useless these are

### CODE REVIEW

**Comment by Xhuis** [comment]
Is making 1-2 line change PRs in the GitHub editor the new trend? #18380

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -388,4 +387,4 @@
 /datum/robot_energy_storage/medical
 	max_energy = 2500
 	recharge_rate = 250
-	name = "Medical Synthesizer"
\ No newline at end of file
+	name = "Medical Synthesizer"
```
Comment (pudl) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: ban this sick filth

**Comment by Cebasiz** [comment]
you should just remove standard borgs tbh

**Comment by KorPhaeron** [comment]
Please explain your PR/provide a fix along with it thanks

**Comment by Cebasiz** [comment]
it's just the reactionary anti-borg camp trying to push through as much bullshit as possible, they've long since stopped caring if what they're doing is going to improve game balance, they've realized they're a "majority" (well, not really) but they have pull in the coder circles and they're trying to jam as much shit through as possible while they have the run of the place.

**Comment by pudl** [comment]
^^^^ butthurt silly con player

**Comment by Cebasiz** [comment]
excellent post my friend, i'll consider the deep meaning there for a few hours and get back to you.

**Comment by Zilenan91** [comment]
It's basically the final death of secborgs. If people are truly that determined, they could go standard borg and stunbaton people and drag them to the brig if they want to be a secborg that bad. HG doesn't want that to happen.

**Comment by KorPhaeron** [comment]
It'd still make way more sense to refactor the borg config and turn off standard borg than to leave a gutted module with a crowbar and fire extinguisher for downstreams to deal with

**Comment by Zilenan91** [comment]
Guess that's done then

**Comment by LatD** [comment]
Standard borgs have eswords when emagged, sad to see them go

**Comment by xxalpha** [comment]
Administration problems being "fixed" by removing features.

**Comment by GunHog** [comment]
This PR should be held as the standard for poor design decisions.

**Comment by AdamElTablawy** [comment]
no matter what you do with 'xxxxd remove hardstuns for borgs' dragging beepsky will always be a thing and as such this really has no impact

even a serviceborg/mediborg can validhunt if they're really that determined (eyedropper of neurotoxin)

the solution is not to remove a cool borg module and instead ban validhunting shitters who can't follow their laws

**Comment by PKPenguin321** [comment]
>dragging beepsky will always be a thing

Nah, it could be removed

**Comment by MMMiracles** [comment]
so at what point are we just going to turn all the other borgs into different variants of eggborgs with no actual way to follow their default laws past bolting down a room and yelling 'HARM'?

**Comment by Malkevin** [comment]
B.b..but its the last borg protecting us from the lizard menace.

**Comment by Armhulen** [comment]
:+1:

**Comment by optimumtact** [comment]
@KorPhaeron @Cheridan needs signoff from design leads

**Comment by KorPhaeron** [comment]
Incoming sent me a guide to making a better config option for borgs, let me try that first

**Comment by xxalpha** [comment]
Feature removal should require merge tokens.

**Comment by Incomptinence** [comment]
Keep the standard module

Man you old age has made you soft granny

**Comment by KorPhaeron** [comment]
#18433 

Since this PR is controverisal have tried a compromise here

**Pull request closed without merging**